setup.cfg

```diff
@@ -19,4 +19,8 @@ known_first_party = uvicorn,tests
 known_third_party = click,does_not_exist,gunicorn,h11,httptools,pytest,requests,setuptools,urllib3,uvloop,watchgod,websockets,wsproto,yaml
 
 [tool:pytest]
-addopts = --cov=uvicorn --cov=tests -rxXs
+addopts = -rxXs
+
+[coverage:run]
+omit=venv/*
+source=uvicorn,tests
```

Comment:
I don't want to be so pedantic, but can you add a newline?

Reply:
done